pkg/reconciler/v1alpha1/service/service.go

```diff
@@ -244,13 +244,16 @@ func (c *Reconciler) reconcile(ctx context.Context, service *v1alpha1.Service) e
 				want[idx].ConfigurationName = ""
 			}
 		}
-		ignoreURLs := cmp.FilterPath(
+		ignoreFields := cmp.FilterPath(
 			func(p cmp.Path) bool {
-				return p.String() == "URL"
+				return p.String() == "TrafficTarget.URL" ||
+					p.String() == "TrafficTarget.Name" ||
```

Comment:
I don't think this field exists?

Reply:
You are right, I fixed this in the mixing bowl